.github/CODEOWNERS

```diff
@@ -41,6 +41,7 @@
 */brave*              @AdmiringWorm
 */nexus-repository*   @chocolatey-community/chocolatey-team-maintainers
 */totalcommander*     @pauby
+*/kubescape*          @chocolatey-community/chocolatey-team-maintainers @HollowMan6
```

Comment:
The `chocolatey-team-maintainers` will not be responsible for this package, please remove that team from this item.

Only those that should be requested to review any PR's/make changes should be listed as the code owner of the packages.

Reply:
Just changed you to be the code owner.